ansible/template-proxies.yml

```diff
@@ -27,6 +28,15 @@
             <Server name="hosted-apis"/>
           </LoadBalancer>
           <Path>/{{ service_id }}{{ '-' + pr_number if pr_number else '' }}</Path>
+    HOSTED_TARGET_CONNECTION_WITH_PATH_REWRITE: |
+          <SSLInfo>
+            <Enabled>true</Enabled>
+          </SSLInfo>
+          <LoadBalancer>
+            <Server name="hosted-apis"/>
+          </LoadBalancer>
+          <Path>/{{ service_id }}{{ '-' + pr_number if pr_number else '' }}/{{fixedpath}}</Path>
```

Comment:
should be `{fixedpath}` (it's an Apigee runtime variable, not a Ansible variable. Hopefully it will appear in the API Proxy as-is. Will have to verify that)

Reply:
That's okay then, was unsure about how the apigee variables were used with ansible. Will remove the lookup for it now.